app/models/registration/campaign.rb

```diff
@@ -0,0 +1,30 @@
+module Registration
+  class Campaign < ApplicationRecord
+    belongs_to :campaignable, polymorphic: true
+
+    has_many :registration_items,
+             class_name: "Registration::Item",
+             dependent: :destroy,
+             inverse_of: :registration_campaign
+
+    has_many :user_registrations,
+             class_name: "Registration::UserRegistration",
+             dependent: :destroy,
+             inverse_of: :registration_campaign
+
+    has_many :registration_policies,
+             class_name: "Registration::Policy",
+             dependent: :destroy,
+             inverse_of: :registration_campaign
+
+    enum :allocation_mode, { first_come_first_serve: 0,
```

Comment:
[nitpick] I only find [this string](https://en.wiktionary.org/wiki/first_come,_first_served) in Wiktionary: "first come, first served" (note the additional "d" at the end). Same in [dict](https://www.dict.cc/?s=First+come+first+serve).

Reply:
I replaced it (but not in the architecture book yet to keep the PR small).